app/src/main/java/com/nextcloud/talk/controllers/ConversationsListController.java

```diff
@@ -857,17 +857,22 @@ protected String getTitle() {
 
     @Override
     public boolean onItemClick(View view, int position) {
-        selectedConversation = ((ConversationItem) Objects.requireNonNull(adapter.getItem(position))).getModel();
-        if (selectedConversation != null && getActivity() != null) {
-            if (showShareToScreen) {
-                handleSharedData();
-                showShareToScreen = false;
-            } else if (forwardMessage) {
-                openConversation(bundle.getString(BundleKeys.INSTANCE.getKEY_FORWARD_MSG_TEXT()));
-                forwardMessage = false;
-            } else {
-                openConversation();
+        try {
+            selectedConversation = ((ConversationItem) Objects.requireNonNull(adapter.getItem(position))).getModel();
+            if (selectedConversation != null && getActivity() != null) {
+                if (showShareToScreen) {
+                    handleSharedData();
+                    showShareToScreen = false;
+                } else if (forwardMessage) {
+                    openConversation(bundle.getString(BundleKeys.INSTANCE.getKEY_FORWARD_MSG_TEXT()));
+                    forwardMessage = false;
+                } else {
+                    openConversation();
+                }
             }
+        } catch (ClassCastException e) {
```

Comment:
auto format kicked in so might be hard to find the fix..
this try catch is the only change for the fix....